control/control.openSUSE.xml

```diff
@@ -143,36 +143,36 @@ textdomain="control"
 
             <!-- Replacement for EXTRAURLS and OPTIONALURLS -->
             <extra_url>
-                <baseurl>http://download.opensuse.org/factory/repo/oss/</baseurl>
+                <baseurl>http://download.opensuse.org/tumbleweed/repo/oss/</baseurl>
                 <alias>repo-oss</alias>
-                <name>openSUSE-Factory-Oss</name>
+                <name>openSUSE-Tumbleweed-Oss</name>
                 <prod_dir>/</prod_dir>
                 <enabled config:type="boolean">true</enabled>
                 <autorefresh config:type="boolean">true</autorefresh>
                 <priority config:type="integer">99</priority>
             </extra_url>
             <extra_url>
-                <baseurl>http://download.opensuse.org/factory/repo/non-oss/</baseurl>
+                <baseurl>http://download.opensuse.org/tumbleweed/repo/non-oss/</baseurl>
                 <alias>repo-non-oss</alias>
-                <name>openSUSE-Factory-Non-Oss</name>
+                <name>openSUSE-Tumbleweed-Non-Oss</name>
                 <prod_dir>/</prod_dir>
                 <enabled config:type="boolean">true</enabled>
                 <autorefresh config:type="boolean">true</autorefresh>
                 <priority config:type="integer">99</priority>
             </extra_url>
             <extra_url>
-                <baseurl>http://download.opensuse.org/debug/factory/repo/oss/</baseurl>
+                <baseurl>http://download.opensuse.org/debug/tumbleweed/repo/oss/</baseurl>
                 <alias>repo-debug</alias>
-                <name>openSUSE-Factory-Debug</name>
+                <name>openSUSE-Tumbleweed-Debug</name>
                 <prod_dir>/</prod_dir>
                 <enabled config:type="boolean">false</enabled>
                 <autorefresh config:type="boolean">true</autorefresh>
                 <priority config:type="integer">99</priority>
             </extra_url>
             <extra_url>
-                <baseurl>http://download.opensuse.org/source/factory/repo/oss/</baseurl>
+                <baseurl>http://download.opensuse.org/source/tumbleweed/repo/oss/</baseurl>
```

Comment:
http://download.opensuse.org/source/tumbleweed/repo/oss/

Object not found!

The requested URL was not found on this server. If you entered the URL manually please check your spelling and try again.

If you think this is a server error, please contact the webmaster.

Error 404

download.opensuse.org
Wed Jan 28 17:57:46 2015
Apache/2.2.12 (Linux/SUSE)